database/testhelpers/test_helpers.go

```diff
@@ -66,9 +74,8 @@ func GetConnectionDetails() *database.Config {
 }
 
 // ConnectToDatabase opens connection to database and returns pointer to instance of database.DB
-func ConnectToDatabase(conn *database.Config) (dbConn *database.Instance, err error) {
+func ConnectToDatabase(conn *database.Config, runMigration bool) (dbConn *database.Instance, err error) {
```

Comment:
tests seem to always run migration. We could drop this?

Reply:
https://github.com/xtda/gocryptotrader/blob/ohlcv-datastore/database/testhelpers/test_helpers_test.go#L75
It is set to false in its own test usage will look to see if i can clean it up

Reply:
I have removed the migration bool from this function no harm in running it during its own test

Reply:
👍 Okay!